license_header.txt

```diff
@@ -1,5 +1,5 @@
 /*!
 Plottable @VERSION (https://github.com/palantir/plottable)
-Copyright 2014-2017 Palantir Technologies
+Copyright 2014-present Palantir Technologies
```

Comment:
Yeah I think "present" is an acceptable thing to put here; @themadcreator @adidahiya ?